css/index.css

```diff
@@ -47,100 +138,206 @@ table {
 	border-spacing: 0;
 }
 
+/* end of styling reset  */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+	/* border: 2px solid black; */
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Roboto', sans-serif;
+/* GENERAL STYLING  */
+html,
+body {
+	height: 100%;
+	font-family: 'Roboto', sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
-    font-family: 'Rubik', sans-serif;
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-size: 18px;
+	margin-bottom: 15px;
+	font-family: 'Rubik', sans-serif;
 }
 
 p {
-    line-height: 1.4;
+	line-height: 1.4;
 }
 
 .container {
-    width: 800px;
-    margin: 0 auto;
+	width: 800px;
+	margin: 0 auto;
 }
 
+a {
+	text-decoration: none;
+	color: black;
+}
+
+/* HEADER */
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	margin: 2% 0;
+}
+
+header nav a {
+	padding: 0 4%;
+	align-self: flex-end;
+}
+
+.header_img {
+	margin: 2% 0;
+}
+
+/* BODY */
+
+/* HOME PAGE  */
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	margin-bottom: 20px;
+	border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+	width: 48%;
+	padding: 0 1%;
+	padding-bottom: 20px;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+	margin-bottom: 20px;
+	border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+	padding: 0 2%;
+	margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+	width: 12.5%;
+	height: 100px;
+	margin: 20px 2.5%;
+	color: white;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
```

Comment:
Nice work using the parent style box to style all of the individual boxes and just overriding the color for each box.